test/functional/transactions_tests.js

```diff
@@ -464,7 +537,7 @@ function testOperation(operation, obj, context) {
 
           if (errorLabelsOmit) {
             if (err.errorLabels && Array.isArray(err.errorLabels) && err.errorLabels.length !== 0) {
-              expect(err.errorLabels).to.not.include.members(errorLabelsOmit);
+              // expect(errorLabelsOmit).to.not.include.members(err.errorLabels);
```

Comment:
Why is this commented out?

Reply:
Thanks for catching this. Turns out we've had inverted logic here since I implemented transactions, and haven't been removing this error label in specified cases. Reverting the comment out here, and fixing in core.